code/modules/projectiles/guns/projectile/revolver/sky_driver.dm

```diff
@@ -36,6 +45,13 @@
 	..()
 
 /obj/item/gun/projectile/revolver/sky_driver/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/device/von_krabin))
+		user.drop_from_inventory(I)
+		qdel(I)
+		name = "Guided S REV .35 Auto \"Sky Driver\""
+		desc = "Old, Syndicate revolver made on lost tech before the Corporate war. Uses .35 Auto rounds. This one has been augmented with psionic-tracking"
```

Comment:
This description looks unfinished

Reply:
Are you sure a wallbanging weapon that can't miss is a good idea?

Reply:
You have to sacrifice an artifact of another faction, that you wouldn't have any use out of, so not particularly balanced. IH also has fucking access to Moelabs and can just simply take the von-krabin, lmao.

Reply:
You don't sacrifice the von-krabin as IH unless moebius really fucked you over.

Reply:
> You have to sacrifice an artifact of another faction, that you wouldn't have any use out of, so not particularly balanced. IH also has fucking access to Moelabs and can just simply take the von-krabin, lmao.

Yeah, it'd cause a massive conflict
And literally everyone can and does use the krab